HW4_Diana_Mokhnitskaya/HW4_task1.py

```diff
@@ -0,0 +1,7 @@
+numbers = [1, 2, 3, 4, 5, 6, 7, 8]
+list_with_index_and_value = tuple(enumerate(numbers))
+
+for even in range(0, 7, 2):
+    print("Even index:" + str(list_with_index_and_value[even]))
+for odd in range(1, 8, 2):
+    print("Odd index:" + str(list_with_index_and_value[odd]))
```

Comment:
bad solution. you hard code indexes in range. NEVER do like this. You solution should work with any list, not only with provided. Please fix

Reply:
Fixed it.